src/main/java/edu/kit/ipd/dbis/controller/GraphEditorController.java

```diff
@@ -176,10 +177,10 @@ public Boolean isValidGraph(PropertyGraph<Integer, Integer> graph) throws Invali
 			statusbar.addMessage(e.getMessage());
 		}
 		if (duplicate) {
-			statusbar.addMessage("Given graph already exists.");
+			statusbar.addMessage("Given graph already exists");
 			this.grapeUI.updateTable();
 		}
-		return duplicate;
+		return !duplicate;
```

Comment:
use more descriptive variable names, I suppose duplicate should actually be named noDuplicate as it is negated.